test/scripts/build-python-image.sh

```diff
@@ -38,5 +38,5 @@ echo "Activating service-account"
 gcloud auth activate-service-account --key-file=${GOOGLE_APPLICATION_CREDENTIALS}
 
 cp $1 Dockerfile
-gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/$2:${VERSION} --project=${PROJECT}
+gcloud builds submit . --tag=${REGISTRY}/${REPO_NAME}/$2:${VERSION} --project=${PROJECT} --timeout=20m
 gcloud container images add-tag --quiet ${REGISTRY}/${REPO_NAME}/$2:${VERSION} ${REGISTRY}/${REPO_NAME}/$2:latest --verbosity=info
```

Comment:
/lgtm
This looks good, but I would like to understand your thoughts on where PyTorch build is taking so long...
cc @Tomcli

Reply:
I believe the PyTorch image is build from Conda source, so it will take around 15-20 minutes depending on the network bandwidth. I think we should cache the first few steps as a separate image so that we don't have to rebuild all the PyTorch libraries and dependencies from conda every time.

Reply:
It may be good way, but I think we can extend the timeout to work around that now, since from the logs, the build almost done when timeout. Thanks @Tomcli